app/enrichment/schemas/header.js

```diff
@@ -19,6 +19,7 @@ module.exports = Joi.object({
   paymentRequestNumber: Joi.number().integer().positive().required(),
   agreementNumber: Joi.string().required(),
   contractNumber: Joi.string().optional(),
+  paymentType: Joi.string().optional(),
```

Comment:
Depends on the outcome of the previous PR, but did suggest this should be an integer.

There is a unit test file that checks the behavior of this schema.  Suggest adding some tests for this property to it.